fastlane/Fastfile

```diff
@@ -179,6 +198,44 @@ platform :ios do
     )
   end
 
+  #####################################################################################
+  # update_jetpack_appstore_strings
+  # -----------------------------------------------------------------------------------
+  # This lane updates the AppStoreStrings.po files for the WordPress app with
+  # the latest content from the release_notes.txt file and the other text
+  # sources
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane update_jetpack_appstore_strings version:<release note version>
+  #
+  # Example:
+  # bundle exec fastlane update_jetpack_appstore_strings version:10.7
+  #####################################################################################
+  desc 'Updates the AppStoreStrings.po file for the WordPress app with the latest data'
+  lane :update_jetpack_appstore_strings do |options|
+    source_metadata_folder = File.join(PROJECT_ROOT_FOLDER, 'fastlane', 'appstoreres', 'jetpack_metadata', 'source')
```

Comment:
I chose this path to stay in line with https://github.com/wordpress-mobile/WordPress-iOS/pull/16571.